packages/cf-component-tabs/src/Tabs.js

```diff
@@ -34,50 +34,50 @@ const TabsGroup = createComponent(
 
 const TabItem = createComponent(
   ({ theme, selected }) => ({
-    flex: theme.flex,
-    background: theme.background,
-    color: theme.color,
-    cursor: theme.cursor,
-    display: theme.display,
-    margin: theme.margin,
-    padding: theme.padding,
-    position: theme.position,
-    textAlign: theme.textAlign,
-    verticalAlign: theme.verticalAlign,
-    outline: theme.outline,
-    userSelect: theme.userSelect,
+    flex: theme.item.flex,
+    background: theme.item.background,
```

Comment:
Just wondering if you know why it did this. Seems interesting. Does it do it to all our components?